Src/Plugins/AzureDevOpsConfiguration/AzureDevOpsConfiguration.csproj

```diff
@@ -11,6 +11,12 @@
     <PackageId>Sarif.PatternMatcher.AzureDevOpsConfiguration</PackageId>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <MSBuildWarningsAsMessages>
+	    $(MSBuildWarningsAsMessages);MSB3277;MSB3030
```

Comment:
>MSBuildWarningsAsMessages [](http://example.com/codeflow?start=7&length=25)

Just a question if this is what we want, or fix the warning.

Reply:
It is not possible to fix either of the sources of the warnings, therefore I need to suppress them.

Reply:
I can't update newtonsoft to eliminate the conflict warning and I can't get rid of the copy commands that sometimes don't copy and give warning but eventually copy.

Reply:
@schlaman-ms Do we have an understanding on why this transient fault is occurring for making copies? I agree with @shaopeng-gh that we should try to fix it, rather than suppress it. Is this a known issue?

Reply:
Transient isn't the correct word here really.
It will always have the failure to copy error, but by the end of the build, in each case for the 4 different warnings, each of those 4 files ends up copied.

The copy parameter for ONLY those 4 has the flag set to continue on error = true. ALL the other copies are continue on error=false.

It looks like those copy commands and continue on error= true settings have not changed for about 2 years. Somebody recognized it needed to be able to continue and that those would have an error but still needed to be there.

Can we have that be a separate task/PR  to investigate about why those copies are like that? I'm guessing that if the continue on error was set to true in the past there was a decent reason for it but who knows.